iceoryx_posh/source/roudi/port_manager.cpp

```diff
@@ -764,7 +764,23 @@ PortManager::acquirePublisherPortData(const capro::ServiceDescription& service,
                                       const RunnableName_t& runnable [[gnu::unused]], // @todo #25 Fix introspection
                                       const PortConfigInfo& portConfigInfo) noexcept
 {
-    /// @todo #25 Add unique check for single producer type - see acquireSenderPortData()
```

Comment:
I would prefer having a boolean member in port manager which is called like `m_hasMultiPublisherSupport` instead of working with macros.
Why are we using macros here?

Or it could be possible to store such a boolean or maybe enum in `iceoryx_posh_types.hpp` as a constexpr.

Reply:
This has to switch the used types during compile time. I'm wondering if we can do it similar to the CMake parameters for MAX_PUBLISHERS etc. What do you think @elfenpiff @elBoberido

Reply:
yes, I already talked with @mossmaurice about that. He is working on it

Reply:
Remove macros